src/content/docs/tutorial-create-alerts/manage-alert-quality.mdx

```diff
@@ -0,0 +1,182 @@
+
+**Goal:** Reduce the number of low value / nuisance incidents.
+
+**Best practices:**
+* Ensure condition settings are intended to detect real business impact.
+* Ensure condition settings are detecting abnormal behavior.
+* Communicate that the incident details `Acknowledge` feature helps measure meaningful and actionable alerts. See [percentage incident acknowledge KPI](#kpi-user-engagement).
+* Report AQM KPIs to all stakeholders.
+
+```sql
+FROM NrAiIncident SELECT count(*) AS 'Incident Count' WHERE event = 'open' AND priority = 'critical' SINCE 1 WEEK AGO COMPARE WITH 1 WEEK AGO
+```
+  </Collapser>
+
+  <Collapser
+    id="kpi-incident-duration"
+    title="Accumulated incident duration KPI"
+  >
+This is the total sum of minutes that all incidents have accumulated over a period of time. Typically you should compare the current and previous weeks.
+
+**Goal:** Reduce the total accumulated minutes of incidents.
+
+**Best practices:**
+* Avoid manually closing incidents that don't motivate some sort of investigatory action. Manual closure will skew the real duration of incident length.
+* Remove alerts that don't result in any remediation actions from the recipients.
+* Improve **percent investigated** and **mean-time-to-investigate** KPIs by communicating their importance in improving detection and response times.
+* Report AQM KPIs to all stakeholders.
+
+```sql
+FROM NrAiIncident SELECT sum(durationSeconds)/60 AS 'Incident Minutes' WHERE event = 'close' AND priority = 'critical' SINCE 1 WEEK AGO COMPARE WITH 1 WEEK AGO
+```
+  </Collapser>
+
+  <Collapser
+    id="kpi-mttc"
+    title="Mean time to close (MTTC) KPI"
+  >
+This is simply the average duration of incidents within the period of time measured. You want this number to be as low as possible.
+
+**Goal:** Reduce MTTC
+
+**Best practices:**
+* Don't manually close incidents. Manual closure will skew the real duration of incident length.
+* Improve reliability engineering skills.
+* Report AQM KPIs to all stakeholders.
+
+```sql
+FROM NrAiIncident SELECT average(durationSeconds/60) AS 'Incident MTTC (minutes)' WHERE event = 'close' AND priority = 'critical' SINCE 1 WEEK AGO COMPARE WITH 1 WEEK AGO
+```
+  </Collapser>
+
+  <Collapser
+    id="kpi-pct-under-five"
+    title="Percent under 5 minutes KPI"
+  >
+This is the percentage of incidents whose total duration is under five minutes. This can be an indicator of of an incident changing state too frequently, which can obscure the cause and severity of the incident. This state is known as **incident flapping**.
+
+**Goal:** Minimize percentage of incidents with short durations.
+
+**Best practices:**
+* Ensure that conditions are detecting legitimate deviations from expected behavior.
+* Understand service level management.
+* Ensure that conditions are detecting legitimate deviations that correlate to business impact or impending business impact.
```

Comment:
`Ensure that conditions are detecting legitimate deviations that correlate to business impact or impending business impact.`

Far too many conceptually complex, multi-syllable words (4.5/100 fleisch-kincaid score https://goodcalculators.com/flesch-kincaid-calculator/)